src/composition.cc

```diff
@@ -116,6 +116,18 @@ Composition::Composition(int prev_decay, ChainPtr decay_line)
   next_id_++;
 }
 
+std::string Composition::ToString(CompMap v) {
+  std::string comp = "{";
+  for (cyclus::CompMap::const_iterator it = v.begin();
+       it != v.end(); ++it) {
+    comp += std::string("{") + std::to_string(it->first) +
+            std::string(",") + std::to_string(it->second) + std::string("},");
```

Comment:
Some space and a different delimiter for readability
```suggestion
            std::string(" : ") + std::to_string(it->second) + std::string("},");
```